api/v1alpha2/gcpcluster_types.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// ClusterFinalizer allows ReconcileGCPCluster to clean up GCP resources associated with GCPCluster before
+	// removing it from the apiserver.
+	ClusterFinalizer = "gcpcluster.infrastructure.cluster.x-k8s.io"
+)
+
+// GCPClusterSpec defines the desired state of GCPCluster
+type GCPClusterSpec struct {
+	// NetworkSpec encapsulates all things related to GCP network.
+	NetworkSpec NetworkSpec `json:"networkSpec,omitempty"`
+
+	// Project is the name of the project to deploy the cluster to.
+	Project string `json:"project"`
+
+	// The GCP Region the cluster lives in.
+	Region string `json:"region"`
+
+	// The default zone to create instances in.
+	// If empty, the first zone available in the Region is used.
+	// +optional
+	DefaultZone *string `json:"defaultZone,omitempty"`
```

Comment:
We probably inherited this from v1alpha1, but it's surprising to see this on the cluster.

Reply:
This should have been removed, will file a PR